packages/adapter/src/lib/adapter/adapter.js

```diff
@@ -9055,12 +9055,13 @@ class AdapterClass extends EventEmitter {
 
                 adapterStates.subscribeLog('system.adapter.' + this.namespace);
             } else {
-                this.requireLog = _isActive => {
+                this.requireLog = _isActive =>
+                    // show warning only by enabling
```

Comment:
as mentioned in Telegram, can we please delay adapter.js changes until the refactoring to ts is done

Reply:
done